crates/namada/src/ledger/native_vp/ibc/context.rs

```diff
@@ -80,25 +81,27 @@ where
     fn read_bytes(&self, key: &Key) -> Result<Option<Vec<u8>>> {
         match self.store.get(key) {
             Some(StorageModification::Write { ref value }) => {
-                let gas = key.len() + value.len();
+                let gas = checked!(key.len() + value.len())? as u64;
```

Comment:
fine not to use a `u64::try_from` as in other places? Same for below

Reply:
I think it's ok to assume 64bit arch - we'll have a warning if not (https://github.com/anoma/namada/pull/3215)

Reply:
also it won't fail on 32bit